lldb/test/API/lang/swift/variables/uninitialized/main.swift

```diff
@@ -16,14 +16,14 @@ struct A<T> {
     }
 
     mutating func a() -> T? {
-        var adict : [String: Any]  //% self.expect("frame variable adict", "Frame variable of an uninitialized dict returns")
+        var adict : [String: Any]  //% self.expect("frame variable adict", substrs=["Frame variable of an uninitialized dict returns"])
```

Comment:
turns out this test was broken

Reply:
I've tried a few git searches (ex `git log -p -S'of an uninitialized'` lldb) and I can't find where this error string would have come from.

Currently, lldb prints an uninitialized dict as an empty value:

```
(lldb) v d
([Int : Int]) d = {}
```

Reply:
for this change, I'm going change the expected string to match what lldb currently does, which is a bit tautological. We can figure out what this should really be doing in a follow up.

Reply:
Originally added in 67f5aeb9ac4f6f1171eed774fd7b09b3b93418c0 by @jimingham. Jim, any thoughts?

Reply:
This wasn't actually my test, I changed how we made the uninitialized dict, but that doesn't much matter.  IIUC, the intent of this test is not to capture how we printed the results of an uninitialized dict, maybe because depending on the context the actual invalid value could differ.  Rather it is just testing that the command succeeded.  

In a self.expect used this way the first string is the command and the second the msg, right?  So this is a test that runs the command and checks that the result is "succeeded" and that's all.   That seems a reasonable thing to do under some circumstances.  But of course you can disambiguate this by adding `msg=` before the message?

Reply:
ah that's my mistake, thanks!

Reply:
My change in [D88792](https://reviews.llvm.org/D88792) broke the scenario you describe. A call like `self.expect("some command", "some message")` is considered a mistake. I made a mistake in following the logic on this, but only one test needed to be changed in llvm, and then there's this. If there's only two tests using `expect` this way, maybe it's ok to not support it.

Reply:
Would it be possible to have your change handle the case where you do:

self.expect("some command", msg="some message")

That would allow you to rule out cases where people were passing a test string in the wrong manner, and still support the ability to say "run this command and assert that it succeeds" which still seems a reasonable thing to do.

Reply:
I'll look to see what we can do. It's supported with python 3 and up, but not sure what the best way to support it is in python2.

Reply:
If you can only do it for Python 3 maybe that's okay, just disable the tests that use this feature when the python version is 2.  If it's just a couple of tests, that's probably okay, Python 2 support isn't going to be with us for long anyway.

Reply:
I made a change that forces callers to use `msg=...` and it results in 258 test failures for my build configuration of mainline llvm, and probably more actual call sites. I'm not sure it's worth changing all those by hand, maybe I can automate it. Then there's the question, would there be enough agreement that requiring `msg=` is a good thing.

Reply:
oh, maybe I can make it so that `msg=` is only require if you're not using any other checks.

Reply:
https://reviews.llvm.org/D91193

Reply:
@Teemperor has made a good point that the `expect("command", "message")` form can be expressed as `runCmd("command", "message")`, and as such `expect` doesn't need to support it. I agree and so I've changed this PR to use `runCmd` in the cases where that's what the call site is intending.